### DISCUSSION TITLE
laybuy payments option requires defining installment

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#531] [Ideas]
Hi,
we need option that laybuy payment (full or partial )  can be done with or without  installements,
amount will be recorded as due ,customer section shows customer balances ,payments and dues amounts

**To Reproduce**
1. Go to 'pay'
2. Click on 'laybuy.'
3. Scroll down to '.....'

**Expected behavior**

**Screenshots**

**Environment:**
 - NexoPOS 4.7
 - Premium Extensions:
 -  Gastro (4.6)
![Screenshot (4)](https://user-images.githubusercontent.com/22215705/150591112-99504052-5417-4258-a8f8-7ddb35498678.png)
.2)]

**Comment by Blair2004** [comment]
This has been moved to discussions as it's pointing out an improvement.
